tests/test_semiology.py

```diff
@@ -14,7 +14,7 @@ def test_neutral_symptoms_side_aphasia(self):
         self.assertIs(type(num_patients_dict), dict)
 
     def test_neutral_dominant_hemisphere_aphasia(self):
-        semiology = Semiology('Aphasia', Laterality.LEFT, Laterality.NEUTRAL)
+        semiology = Semiology('Aphasia', Laterality.NEUTRAL, Laterality.NEUTRAL)
```

Comment:
As above

Reply:
because these cases won't exist with these semiology: laterality attribute will always default to None?